server/routes/elasticsearch.js

```diff
@@ -39,9 +39,16 @@ export default function(server, services) {
     method: 'POST',
     handler: elasticsearchService.getMappings,
   });
+
   server.route({
     path: '/api/alerting/_plugins',
     method: 'GET',
     handler: elasticsearchService.getPlugins,
   });
+
+  server.route({
+    path: '/api/alerting/_settings',
```

Comment:
Was there not an already existing kibana endpoint to get cluster settings? I'd assume somewhere in Kibana is already doing it.

Reply:
If you mean an endpoint defined by us for use in the Alerting plugin, I didn't find anything.

The underlying call is to `await callWithRequest(req, 'cluster.getSettings')` which wasn't being used anywhere in the Kibana plugin until these changes.